gateway/.envs/example/jupyterhub.env.example

```diff
@@ -0,0 +1,64 @@
+# JupyterHub Production Environment Configuration
+# Copy this file to jupyterhub.env and configure for your production environment
+
+# =============================================================================
+# AUTH0 OAUTH CONFIGURATION
+# =============================================================================
+# Get these values from your Auth0 dashboard
+AUTH0_DOMAIN=your-domain.auth0.com
+AUTH0_CLIENT_ID=your_client_id_here
+AUTH0_CLIENT_SECRET=your_client_secret_here
+
+# =============================================================================
+# JUPYTERHUB CONFIGURATION
+# =============================================================================
+# JupyterHub host URL (must match Auth0 callback URL)
+JUPYTERHUB_HOST=sds.crc.nd.edu/notebook
+
+# Cookie secret for session management (generate a secure random string)
+JUPYTERHUB_CRYPT_KEY=your_64_character_hex_secret_here
+
+# =============================================================================
+# USER ACCESS CONTROL
+# =============================================================================
+# Option 1: Allow specific users (emails)
+# Comma-separated list of allowed users
+JUPYTERHUB_ALLOWED_USERS=user1@domain.com,user2@domain.com,user3@domain.com
+
+# Option 2: Allow users by Auth0 groups (recommended for large organizations)
+# Comma-separated list of allowed groups
+# JUPYTERHUB_ALLOWED_GROUPS=jupyter-users,data-scientists,researchers
+
+# Comma-separated list of admin users (emails)
+JUPYTERHUB_ADMIN_USERS=admin@domain.com,admin2@domain.com
+
+# Note: If both ALLOWED_USERS and ALLOWED_GROUPS are empty,
+# all Auth0 users will be allowed (NOT recommended for production)
+
+# =============================================================================
+# DOCKER CONFIGURATION
+# =============================================================================
+# User configuration for containers
+NB_UID=1000
+NB_GID=100
+DOCKER_GID=103
+
+# Docker network and image configuration
+DOCKER_NETWORK_NAME=gateway_jupyter-net
+DOCKER_NOTEBOOK_IMAGE=quay.io/jupyter/base-notebook:latest
+DOCKER_NOTEBOOK_DIR=/home/jovyan/work
+
+# =============================================================================
+# SECURITY NOTES
+# =============================================================================
+# 1. Generate a secure cookie secret:
+#    python3 -c "import secrets; print(secrets.token_hex(32))"
+#
+# 2. Configure Auth0 application:
+#    - Set callback URL to: https://sds.crc.nd.edu/notebook/hub/oauth_callback
```

Comment:
let's use `https://example.com/notebook/hub/oauth_callback` to better match other deployments of SDS